lib/config/executor/container_vm.go

```diff
@@ -44,6 +44,21 @@ type Common struct {
 	Notes string `vic:"0.1" scope:"hidden" key:"notes"`
 }
 
+// Common data (specifically for a containerVM) between managed entities, across execution environments.
+type CommonSpecForVM struct {
```

Comment:
It's also exactly the same as `Common`. It's copy and pasted. @chengwang86 just use the original object.